src/client/pages/Embark/index.tsx

```diff
@@ -151,7 +159,13 @@ const Embark: React.FunctionComponent<EmbarkProps> = (props) => {
                 passage={currentPassage}
                 storyData={state.data}
                 startPageLink={props.startPageLink}
-                customTrailingContent={<LanguagePicker />}
+                customTrailingContent={
+                  market === Market.Se ? (
+                    <PhoneNumber color="white" />
+                  ) : (
+                    <LanguagePicker />
+                  )
+                }
```

Comment:
Related to the comment @robinandeer made that I also answered:
So instead of using the `useMarket` that's actually deprecated and shouldn't be used (I've just failed to communicate this properly) I suggest importing `useCurrentLocale` from `l10n/useCurrentLocale.ts` and then do
```suggestion
                customTrailingContent={
                  currentLocale.phoneNumber ? (
                    <PhoneNumber color="white" />
                  ) : (
                    <LanguagePicker />
                  )
                }
```
Same goes for the other files where markets are checked to see if the phone number should show 😊